spaceship/lib/spaceship/test_flight/group.rb

```diff
@@ -84,15 +84,15 @@ def self.perform_for_groups_in_app(app: nil, groups: nil, &block)
       if groups.nil?
         default_external_group = app.default_external_group
         if default_external_group.nil?
-          UI.user_error!("The app #{app.name} does not have a default external group. Please make sure to pass group names to the `:groups` option.")
+          raise "The app #{app.name} does not have a default external group. Please make sure to pass group names to the `:groups` option."
```

Comment:
My feeling is that we need to be sure that we surface this failure as a user-error of some sort up the chain. Failing to include a group name or mistyping a group name (as in line 95 below) is not really a crash, yet if we do not rescue this error and re-raise as a user error, it will be logged as a crash by _fastlane_'s `runner.rb`.